source/Jobbr.Server/TinyMessenger.cs

```diff
@@ -1,5 +1,4 @@
-﻿// <auto-generated />
```

Comment:
Was this generated automatically? If so, should we really be manually changing it?

Reply:
If you copy pasted it can you please leave the auto-generated tag, so we're not reformatting it, as it's essentially "external" code?